apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -54,7 +72,22 @@ export function RequestCourse({
         })}
         <main id="content" class="container">
           <h1 class="sr-only">Request a Course</h1>
-          ${CourseRequestsCard({ rows })}
+          ${CourseRequestsCard({ rows })} ${EncodedData(lti13Info, 'courseRequestLti13Info')}
+          ${Modal({
+            id: 'lti13FillModal',
+            title: html`Auto-fill with ${lti13Info?.['cr-institution'] ?? 'LMS'} data?`.toString(),
+            body: html`<p>
+                You appear to be coming from a course in another learning system. Should we
+                partially fill in this request form with information from that course?
+              </p>
+              <p>(You can edit it after it's auto-filled.)</p>`,
+            footer: html`
+              <button type="button" class="btn btn-success" id="fillCourseRequestLti13">
+                Fill from LMS data
+              </button>
+              <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
```

Comment:
I would not label this "Cancel"; that could be interpreted as canceling the entire course request process.

Reply:
Switched to Ignore in 0a22979fe